.ci/ovn-kubernetes/Dockerfile

```diff
@@ -0,0 +1,78 @@
+FROM fedora:33 AS ovnbuilder
+
+USER root
+
+ENV PYTHONDONTWRITEBYTECODE yes
+
+# install needed rpms - openvswitch must be 2.10.4 or higher
+RUN INSTALL_PKGS=" \
+    python3-pyyaml bind-utils procps-ng openssl numactl-libs firewalld-filesystem \
+    libpcap hostname \
+    python3-openvswitch python3-pyOpenSSL \
+    autoconf automake libtool g++ gcc fedora-packager rpmdevtools \
+    unbound unbound-devel groff python3-sphinx graphviz openssl openssl-devel \
+    checkpolicy libcap-ng-devel selinux-policy-devel" && \
+    dnf install --best --refresh -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
+    dnf clean all && rm -rf /var/cache/dnf/*
+
+# Build OVS and OVN rpms from current folter
+RUN mkdir /tmp/ovn
+COPY . /tmp/ovn
+WORKDIR /tmp/ovn/ovs
+
+RUN ./boot.sh
+RUN ./configure -v
+RUN make && make rpm-fedora
+RUN rm rpm/rpmbuild/RPMS/x86_64/*debug*
+RUN rm rpm/rpmbuild/RPMS/x86_64/*devel*
+
+WORKDIR /tmp/ovn
+RUN ./boot.sh
+RUN ./configure
+RUN make && make rpm-fedora
+RUN rm rpm/rpmbuild/RPMS/x86_64/*debug*
+RUN rm rpm/rpmbuild/RPMS/x86_64/*docker*
+
+# Build ovn-kubernetes
+FROM golang:1.16 as ovnkubebuilder
+#Clone OVN Source Code
```

Comment:
Nit: missing space after `#`, also this comment is a bit confusing, we're actually cloning `ovn-kubernetes`, not `ovn`.  I think we can skip the comment altogether.

Reply:
done